The_Lost_Star/events/activities/expedition_activity/expedition_events.txt

```diff
@@ -0,0 +1,439 @@
+				key = expedition_planetarium_rousing_speech
+				score = 25
+				tags = { random good }
+				character = root
+
+				#Effect
+				root = {
+					duel = {
+						skill = diplomacy
+						value = 16
+						50 = {
+							desc = expedition.4001.d.success
+							compare_modifier = {
+								value = scope:duel_value
+								multiplier = 3.5
+								min = -25
+							}
+							send_interface_toast = {
+								title = expedition.4001.d.success
+								add_gold = minor_gold_value
+								add_prestige = medium_prestige_gain
+								expedition_activity_success_change_effect = { CHANGE = increase_minor }
+							}
+						}
+						40 = {
+							desc = expedition.4001.d.failure
+							compare_modifier = {
+								value = scope:duel_value
+								multiplier = -3.5
+								min = -25
+							}
+							send_interface_toast = {
+								title = expedition.4001.d.failure
+							}
+						}
+					}
+				}
+			}
+		}
+		ai_chance = {
+			base = 70
+			ai_value_modifier = {
+				ai_sociability = 1
+				ai_energy = 0.5
+			}
+		}
+	}
+}
+
+expedition.4002 = {
+	type = activity_event
+	title = expedition.4002.t
+	desc = {
+		desc = expedition.4002.opening
+		desc = expedition.4002.opening.2
+	}
+	theme = expedition_activity_warehouse
+	left_portrait = {
+		character = root
+		animation = thinking
+	}
+	cooldown = { years = 1 }
+
+	option = { #Do nothing
+		name = expedition.4002.a
+		scope:activity = {
+			add_activity_log_entry = {
+				key = expedition_warehouse_do_nothing
+				score = 25
+				tags = { random }
+				character = root
+
+				#Effect
+			}
+		}
+		ai_chance = {
+			base = 30
+			ai_value_modifier = {
+			}
+		}
+	}
+
+	option = { #Search for valuables
+		name = expedition.4002.b
+		scope:activity = {
+			add_activity_log_entry = {
+				key = expedition_warehouse_search_valuables
+				score = 25
+				tags = { random good }
+				character = root
+
+				#Effect
+				root = {
+					duel = {
+						skill = stewardship
+						value = 16
+						50 = {
+							desc = expedition.4002.b.success
+							compare_modifier = {
+								value = scope:duel_value
+								multiplier = 3.5
+								min = -25
+							}
+							send_interface_toast = {
+								title = expedition.4002.b.success
+								add_gold = medium_gold_value
+							}
+						}
+						40 = {
+							desc = expedition.4002.b.failure
+							compare_modifier = {
+								value = scope:duel_value
+								multiplier = -3.5
+								min = -25
+							}
+							send_interface_toast = {
+								title = expedition.4002.b.failure
+							}
+						}
+					}
+				}
+			}
+		}
+		ai_chance = {
+			base = 30
+			ai_value_modifier = {
+				ai_greed = 1
+			}
+		}
+	}
+	
+	option = { #Search for information
+		name = expedition.4002.c
+		scope:activity = {
+			add_activity_log_entry = {
+				key = expedition_warehouse_search_information
+				score = 25
+				tags = { random good }
+				character = root
+
+				#Effect
+				root = {
+					duel = {
+						skill = stewardship
+						value = 16
```

Comment:
Try and use script values, for this it would be 
high_skill_rating (which is 15)
or
very_high_skill_rating (18)

You can find these in "common\script_values\00_basic_values.txt" from CK3